### ISSUE TITLE
[BUG] Player toggling fullscreen mode when switching sources 

### ISSUE THREAD

**Issue by kxmatejka** [jhomlala/betterplayer#466]
**Describe the bug**
When a source is changed then fullscreen mode switches from previous value. Viz screenshot section.

**Example code**

```dart
import 'dart:async';
import 'package:flutter/material.dart';
import 'package:better_player/better_player.dart';

class BetterPlayerFullscreenIssue extends StatefulWidget {
  createState() => _BetterPlayerFullscreenIssueState();
}

class _BetterPlayerFullscreenIssueState extends State<BetterPlayerFullscreenIssue> {
  final List<String> sources = [
    'https://firebasestorage.googleapis.com/v0/b/tivio-dev.appspot.com/o/assets%2Fstatic%2Fad1.mp4?alt=media',
    'https://firebasestorage.googleapis.com/v0/b/tivio-dev.appspot.com/o/assets%2Fstatic%2Fad2.mp4?alt=media',
    'https://firebasestorage.googleapis.com/v0/b/tivio-dev.appspot.com/o/assets%2Fstatic%2Fad3.mp4?alt=media',
    'https://firebasestorage.googleapis.com/v0/b/tivio-dev.appspot.com/o/assets%2Fstatic%2Fad4.mp4?alt=media',
    'https://firebasestorage.googleapis.com/v0/b/tivio-dev.appspot.com/o/assets%2Fstatic%2Fad5.mp4?alt=media',
  ];
  int lastSourceIndex = 0;
  BetterPlayerController betterPlayerController;
  Timer timer;

  @override
  void initState() {
    super.initState();
    startChangingSources();
  }

  startChangingSources() {
    changeSource();
    timer = Timer.periodic(Duration(seconds: 5), (timer) => changeSource());
  }

  changeSource() {
    if (betterPlayerController != null) {
      betterPlayerController.pause();
    }

    var config = BetterPlayerConfiguration(
      aspectRatio: 16 / 9,
      autoPlay: true,
      fullScreenByDefault: true,
      controlsConfiguration: BetterPlayerControlsConfiguration(
        playerTheme: BetterPlayerTheme.material,
      ),
    );

    var source = BetterPlayerDataSource(
      BetterPlayerDataSourceType.network,
      sources[lastSourceIndex],
    );

    setState(() {
      betterPlayerController = BetterPlayerController(config);
      betterPlayerController.setupDataSource(source);

      lastSourceIndex = (lastSourceIndex >= sources.length - 1) ? 0 : lastSourceIndex + 1;
    });
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Container(
        child: (betterPlayerController != null)
            ? BetterPlayer(
                controller: betterPlayerController,
              )
            : null,
      ),
    );
  }

  @override
  void dispose() {
    timer.cancel();
    betterPlayerController = null;
    super.dispose();
  }
}
```

**Expected behavior**
Keep video playing in fullscreen.

**Screenshots**
[demo](https://firebasestorage.googleapis.com/v0/b/tivio-dev.appspot.com/o/Peek%202021-05-04%2010-49.gif?alt=media&token=1e1ecdf8-e404-44ce-8186-dfce0f0e40fd)

**Flutter doctor**
```
[✓] Flutter (Channel stable, 2.0.4, on Linux, locale en_US.UTF-8)
    • Flutter version 2.0.4 at /home/ubuntu/snap/flutter/common/flutter
    • Framework revision b1395592de (5 weeks ago), 2021-04-01 14:25:01 -0700
    • Engine revision 2dce47073a
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /home/ubuntu/Android/Sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /home/ubuntu/Android/Sdk
    • Java binary at: /usr/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-Ubuntu-0ubuntu2.20.04)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/linux#android-setup for detailed instructions).

[✓] IntelliJ IDEA Ultimate Edition (version 2020.3)
    • IntelliJ at /home/ubuntu/.local/share/JetBrains/Toolbox/apps/IDEA-U/ch-0/203.7148.57
    • Flutter plugin version 56.0.3
    • Dart plugin version 203.8173

[✓] Connected device (2 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86    • Android 10 (API 29) (emulator)
    • Chrome (web)                       • chrome        • web-javascript • Google Chrome 90.0.4430.93

! Doctor found issues in 1 category.
```

**Better Player version**
- Version: 0.0.65

**Comment by jhomlala** [comment]
This will be fixed with next release.

**Comment by jhomlala** [comment]
Released in 0.0.66.
